package.json

```diff
@@ -62,7 +62,6 @@
             "git add"
         ]
     },
-    "proxy": "http://localhost:8080",
```

Comment:
I think this was here if someone decided to use the backend locally, maybe it's worth leaving in?

Reply:
I didn't fully read the pull request description... I didn't seem to be having that issue but it's probably fine removing.